### PR TITLE
Add UI Acceptance / E2E tests 

### PR DESCRIPTION
**Pull request by djejaquino**
## Description

Fixes A11y and internationalization issues.
Adds UI acceptance and E2E tests.

## Testing required outside of automated testing?

- [x] Not Applicable

### Screenshots (if appropriate):

- [x] Not Applicable

## Rollback / Rollforward Procedure

- [x] Roll Forward
- [ ] Roll Back

## Reviewer Checklist

- [ ] Description of Change
- [ ] Description of outside testing if applicable.
- [ ] Description of Roll Forward / Backward Procedure
- [ ] Documentation updated for Change

### CODE REVIEW

File: cypress/fixtures/example.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
```
Comment (koalaty-code) [inline cypress/fixtures/example.json:1]: Is there a way to stop this from being created when running Cypress or do we need to add it to our .gitignore?

File: cypress/specs/acceptance/pages/view.spec.ts [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+  });
+
+  it('should handle 500 status code response from the API', () => {
+    // set intercept before the page visit
+    cy.intercept('GET', `/api/secrets/${id}/details`, {
+      statusCode: 500,
+    });
+    visit();
+    // temporarily disable halting on uncaught client exceptions (used to redirect)
+    cy.on('uncaught:exception', () => false);
```
Comment (koalaty-code) [inline cypress/specs/acceptance/pages/view.spec.ts:48]: Does this not need to be before the call to `visit()` which generates the uncaught client exception?

File: package.json [hunk 3]
```diff
@@ -60,15 +60,19 @@
   "devDependencies": {
     "@basis-theory/eslint-config": "^1.0.9",
     "@hapi/iron": "^6.0.0",
+    "@types/chance": "^1.1.3",
+    "@types/jest": "^27.5.0",
```
Comment (koalaty-code) [inline package.json:64]: Do we need this dependency?

File: cypress/fixtures/example.json (hunk 1 above)
Comment (djejaquino) [reply cypress/fixtures/example.json:1]: Setting the fixtures folder should do the trick

File: cypress/specs/acceptance/pages/view.spec.ts (hunk 2 above)
Comment (djejaquino) [reply cypress/specs/acceptance/pages/view.spec.ts:48]: It is safer to move before the `visit` statement. The exception is thrown by the `useApi` hook inside the page, that only runs after the page is completely loaded.

**Review by koalaty-code — APPROVED** [review]

**Pull request merged**